packages/store/test/perf-patterns.js

```diff
@@ -0,0 +1,259 @@
+// @ts-check
+import '@agoric/swingset-vat/tools/prepare-test-env.js';
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { makeTagged } from '@endo/marshal';
+import {
+  AmountMath,
+  makeIssuerKit,
+  AmountShape,
+  BrandShape,
+  PaymentShape,
+} from '@agoric/ertp';
```

Comment:
copies these shapes into the test os we don't complicate layering.

Reply:
Done